app/Http/Controllers/MovieController.php

```diff
@@ -2,9 +2,74 @@
 
 namespace App\Http\Controllers;
 
+use Carbon\Carbon;
+use App\Models\Movie;
 use Illuminate\Http\Request;
 
 class MovieController extends Controller
 {
-    //
+    public function index() //print all data in Moives database
+    {
+        return view('movies.index', [
+            'movies' => Movie::all()
```

Comment:
It is okay to return all, but we usually want to make sure that the data is presented in a useful manner. 
To identify these here is are some easy questions that will help you out:
- What information do I want to show on the index page?
- How should it be ordered?
- What about if we had a million movies? How would I show the records? Would it be paginated?

Reply:
Change the request

It now paginates 5 movies per page

It will also pass in the movies by order of name first (ascending order) and then release date (descending order)